src/layer/marker/Icon.js

```diff
@@ -136,9 +136,10 @@ export var Icon = Class.extend({
 		}
 	},
 
-	_createImg: function (src, el) {
+	_createImg: function (src, el, alt) {
 		el = el || document.createElement('img');
 		el.src = src;
+		el.alt = alt || '';

```

Comment:
> In other part you use 'Location' string instead of empty string.

_Originally posted by @johnd0e in https://github.com/Leaflet/Leaflet/pull/6248#r412839207_

Reply:
> It depends on the use case.
> 
> in this case, you create a new image tag so you need to add alt value or leave it with an empty string.

_Originally posted by @Muhnad in https://github.com/Leaflet/Leaflet/pull/6248#r414186918_